source/features/search-releases.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+
+import features from '../libs/features';
+import {getOwnerAndRepo} from '../libs/utils';
+import {octoface} from '../libs/icons';
+
+async function init() {
+	if (!hasAnyReleases()) {
+		console.debug('The repo does not have any releases.');
```

Comment:
This is temporary right? Either way it can be dropped, we don’t have special debugging for each feature.

Reply:
Yeah. Removed.